### PR TITLE
Update Contributors Section in README

### PR DESCRIPTION
**Pull request by shubhagarwal1**
## 📋 Description
This PR updates the contributors section in the README file to accurately reflect the current contributors to the project. It ensures that all contributions are properly acknowledged, promoting a more inclusive environment.

## 🔨 Changes Made
Removed the update contributor file 

- 
- 
- 

## ✅ Checklist

Before submitting the PR, please make sure you have completed the following:

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation (if applicable).
- [ ] My changes generate no new warnings.
- [ ] I have tested it locally and it works fine.
- [ ] Any dependent changes have been merged and published in downstream modules.

## 🏷️ Types of Changes

What type of changes does your code introduce? (Check all that apply)

- [ ] Bug fix (non-breaking change which fixes an issue) 🐛
- [ ] New feature (non-breaking change which adds functionality) ✨
- [ ] UI enhancement (non-breaking change which enhances UI) 🎨
- [ ] Documentation update 📚

## 🤝 Related Issues

- Fixes #

## 📸 Screenshots (if applicable)

---

## Reminder:
Please ensure your pull request is made to the `test` branch, not `main`.

*Thank you for contributing to VigneshDevHub! We look forward to reviewing your PR and getting it merged!*

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -5287,4 +5287,4 @@
       }
     }
   }
-}
+}
```
Comment (Vignesh025) [inline package-lock.json:5290]: Please don't make any changes in package-lock.json

File: package.json [hunk 2]
```diff
@@ -31,4 +31,4 @@
     "passport-local-mongoose": "^8.0.0",
     "sanitize-html": "^2.13.0"
   }
-}
+}
```
Comment (Vignesh025) [inline package.json:34]: Please don't make any changes in package.json

Comment (shubhagarwal1) [reply package.json:34]: yes yes

File: package-lock.json (hunk 1 above)
Comment (shubhagarwal1) [reply package-lock.json:5290]: appologies for that, pretier extension just make it don this

**Comment by Vignesh025** [comment]
*Thankyou for Contributing!*

**Pull request merged**